notebooks/output-sanitize.cfg

```diff
@@ -71,17 +71,17 @@ replace: 100% Done |  1.0s
 
 [travis-ci_wps_output_url]
 # output_netcdf='http://localhost:5000/outputs/50c0a3f8-67c7-11ea-9e2d-b06ebf31cced/frost-days_SRES-A2-experiment_20460101-20650101.nc
-regex: http://localhost:\d+/outputs/(?:(?:finch/)|(?:weaver/))?
+regex: http://localhost:\d+/outputs/(?:[a-z-_]*/)?
```

Comment:
oh dash being a special char, I think it might need to be escaped.

Reply:
If this is pure Python regex, I think it doesn't need to be escaped,  but it's less confusing if it is!